pkg/performanceprofile/controller/performanceprofile/components/utils.go

```diff
@@ -88,8 +88,16 @@ func (c *CPULists) Intersect() []int {
 }
 
 // CountIsolated returns how many isolated cpus where specified
-func (c *CPULists) CountIsolated() int {
-	return c.isolated.Size()
+func (c *CPULists) CountCPUs(cpus cpuset.CPUSet) int {
```

Comment:
If we use the `Empty` method of `cpuset.CPUSet` (and we should be able to) this method becomes unnecessary.
moreover, this method does not use any field of `CPULists`, hence should have been a free function

Reply:
You are right, much cleaner now